packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -67,10 +67,17 @@
           "src": {
```

Comment:
Could you `oneOf` this instead? With this current schema we could have `{ replaceWith: "foo" }` which is clearly invalid.

Reply:
Yeah that is better.